modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java

```diff
@@ -256,8 +255,16 @@ public String getTitle() throws PortalException {
 			_infoItemServiceTracker.getFirstInfoItemService(
 				InfoItemLanguagesProvider.class, _className);
 
-		Stream<String> stream = Arrays.stream(
-			infoItemLanguagesProvider.getAvailableLanguageIds(_model));
+		List<String> languageIds = new ArrayList<>();
+
+		for (Object model : _models) {
+			_intersect(
+				languageIds,
+				Arrays.asList(
+					infoItemLanguagesProvider.getAvailableLanguageIds(model)));
```

Comment:
Can happens to be a no intersect languages for the different models?

Reply:
The site default locale is always added to the set, so even if the intersection is empty, the returned set of languages will have at least one element.